multi-arch/samples.md

```diff
@@ -0,0 +1,157 @@
+ - and by extension, always provides valid ClusterOperator statuses during install and upgrade, as this amounts to a no-op on those platforms
+
+And in case the implication is not clear, OpenShift development will not be developing or maintaining 
+any non-x86 images for these existing samples.
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+As part of achieving what is described in the opening section, the samples operator needs
+to achieve the desired results within the context of how multi-arch is being achieved for 
+the entire product.
+
+Details around how mulit-arch will be achieved across the entire product are still pending,
+at least at the time of this writing for the author.
+
+In particular:
+ - will support across the entire product be staged along homogoneous / heterogeneous lines
+ - if "homogeneous only" becomes a valid option for the released product, is use of Golang's APIs for determining architecture sufficient
+ - if "heterogenous" is an option, the assumption is the list of valid architectures will be provided in a cluster-wide API object, and that samples operator installs for all architectures listed
+ - if "heterogenous" is an option, some notion outside the scope of the cluster operator API determines if a given sample runs on its associated architecture (for example, is a node selector needed on a deployment config or build leveraging a non-x86 imagestream to make sure the execution occurs on the right architecture?)
+
+## Motivation
+
+Multi-arch has been declared a key initiative for v4.x.
+
+### Goals
+
+x86 samples are not installed in a cluster with no x86 nodes.  Nor are any future non-x86 samples installed in a cluster with only x86 nodes.
+
+### Non-Goals
+
+We do not take on porting existing samples to non-x86 architectures.
+
+## Proposal
+
+The samples operator config object already supports the specification of multiple 
+architectures.
+
+It currently requires that only a single architecture is specified, and that 
+it is x86_64.
+
+In conjunction, the import of content from https://github.com/openshift/library supports 
+downloading multiple architectures, and segregating multiple architectures.  And 
+the tagging in https://github.com/openshift/library allows providers to designate which
+samples go to which platforms.
+
+So enabling the above to allow PPC and Z content if it is available is item one.
+
+But as previously stated, there is currently no such content, so the samples operator
+should do nothing when it comes up in Managed state, and report Available=true, Progressing=false,
+Degraged=false.
+
+We also need to ensure that marking the samples operator as Unmanaged or Removed 
```

Comment:
What do we need to ensure about this?  Dangling sentence?

Reply:
removed in next push .... wasn't anything more needed there